### PR TITLE
Bug/coverage test

### PR DESCRIPTION
**Pull request by soulee-dev**

### CODE REVIEW

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -1,34 +1,47 @@
...
-        SMOKESHOW_GITHUB_COVERAGE_THRESHOLD: 50
-        SMOKESHOW_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        SMOKESHOW_GITHUB_PR_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
```
Comment (sean-k1) [inline .github/workflows/coverage.yml:18]: better use @v4

Comment (soulee-dev) [reply .github/workflows/coverage.yml]: Per your request, I updated `actions/checkout@v2` into v4, and also update `actions/setup-python@v2` into v4

**Comment by soulee-dev** [comment]
We should re-run the actions to test it works.

**Comment by sean-k1** [comment]
@soulee-dev  I re-run action!
Thanks for hard work!

**Review by sean-k1 — APPROVED** [review]

**Pull request merged**